library/ceph_pool.py

```diff
@@ -34,36 +34,36 @@
 """
 
 import time
+import os
+import pickle
```

Comment:
Is there a reason we want to use pickle here?  Seems it would be easier to use a more standard serialization format (ie. json, yaml, etc).  This format would make debugging far more simple.